src/PastJobs/PastJobs.js

```diff
@@ -1,28 +1,26 @@
 import React, { Component } from "react";
 import { Header } from 'react-native-elements';
-import { StyleSheet, Text, SafeAreaView, ScrollView, Picker, View } from 'react-native';
+import { Dimensions, SectionList, StyleSheet, Text, SafeAreaView, ScrollView, Picker, View, FlatList, TouchableOpacity } from 'react-native';
 import Constants from 'expo-constants';
 import { Entypo } from "@expo/vector-icons";
 import { AuthContext } from "../../AuthContext";
 import { db } from '../../config';
 import { sortBy } from 'lodash';
 import { formatTime } from "../RequestorPages/NewJobPage";
 
-
-
 const today = new Date();
 var activeUser  = {
   username: '',
   trustedUsers: [],
 };
 
+const monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
 
-const Job = ({job: {title, jobType, startDateTime, endDateTime, location, requestor, volunteer}}) => {
+const Job = ({job}) => {
```

Comment:
Why get rid of the named parameters?

Reply:
This was a result of me trying to figure out how to pass in the args of the data, but since all of that was unsuccessful, I'll switch it back to the original form so we have it the original way it was.